TappNetwork/home-visiting-app-web
# Cu 2rjyt6n ir sharing

# Details

This PR encompasses changes for the front and backend for sharing resources between visitors. Users may share resources with other users, and the receiver will see the resource on their dashboard and in their share inbox. 

# Steps to test changes

1. Login to front end
2. Check dashboard to see shares section
3. Click on share to open share view modal
4. click exit to exit modal
5. click share inbox to view share inbox
6. ensure your unviewed shares have green 'new' tag
7. click on share sent to you to view modal
8. click delete 
9. ensure share is deleted
10. test filtering
11. go to resource center
12. click share icon to launch share create modal
13. complete form and ensure success in network tab or in DB

# Issues Addressed

[2rjyt6n](https://app.clickup.com/t/2rjyt6n)

# Screenshots

<img width="1072" alt="Screen Shot 2022-09-01 at 11 31 34 AM" src="https://user-images.githubusercontent.com/29612767/187976975-ebd69c34-99aa-41a1-a402-bc14d4ce9020.png">
<img width="1085" alt="Screen Shot 2022-09-01 at 11 31 51 AM" src="https://user-images.githubusercontent.com/29612767/187976988-7b721c2e-8967-4f56-931d-cb2d202f7b56.png">
<img width="979" alt="Screen Shot 2022-09-01 at 11 32 10 AM" src="https://user-images.githubusercontent.com/29612767/187977018-9c29a174-4d24-468f-9dc3-fe674257855f.png">
<img width="1084" alt="Screen Shot 2022-09-01 at 11 32 27 AM" src="https://user-images.githubusercontent.com/29612767/187977040-22375760-5926-405a-a4bf-183748b58af4.png">
<img width="1082" alt="Screen Shot 2022-09-01 at 11 32 35 AM" src="https://user-images.githubusercontent.com/29612767/187977059-bf342cc9-8eb6-41fa-a8d2-72c9fd44ed82.png">
